JsonClient/JsonClient/Program.cs

```diff
@@ -63,102 +72,106 @@ static async Task Main(string[] args)
                 {
                     case 1:
                        var results = await client.GetAsync();
-                       if(results != null)
-                       {
-                        foreach(TObject res in results){
-                               PrintObject(res);
-                            }
-                       }else
-                       {
-                           Console.ForegroundColor = ConsoleColor.Red;
-                           Console.WriteLine("not found");
-                           Console.ResetColor();
-                       }
+                       ManageResult(results,"get");
                         break;
                     case 2:
                         result = await client.GetAsync(GetId());
-                        if(result != null){
-                            PrintObject(result);
-                        }else
-                        {
-                            Console.ForegroundColor = ConsoleColor.Red;
-                            Console.WriteLine("not found");
-                            Console.ResetColor();
-                        }
-
+                        ManageResult(result, "get");
                         break;
                     case 3:
-                       result = await client.PostAsync( GetObject<TObject>(obj));
-                       if(result != null){
-                           PrintObject(result);
-                       }else
-                       {
-                          Console.ForegroundColor = ConsoleColor.Red;
-                          Console.WriteLine("error while Saved");
-                          Console.ResetColor();
-                       }
+                       result = await client.PostAsync( BuildObjectByConsole<TObject>(obj));
+                       ManageResult(result, "post");
                         break;
                     case 4:
-                       result = await client.UpdateAsync( GetId(), GetObject<TObject>(obj));
-                       if(result != null){
-                           PrintObject(result);
-                       }else
-                       {
-                          Console.ForegroundColor = ConsoleColor.Red;
-                          Console.WriteLine("error while updated");
-                          Console.ResetColor();
-                       }
+                       result = await client.UpdateAsync( GetId(), BuildObjectByConsole<TObject>(obj));
+                       ManageResult(result, "update");
                         break;        
                     default:
                         Console.WriteLine("Default case");
                         break;
                 }
                 break;
 
-               }
+            }
+        }
+
+        public static void ManageResult(object result, string NameMethod){

```

Comment:
Replace string NameMethod for an existing enum or create a new one with HTTPMethods